{"name": "测试用例描述：错误的验证码登录失败", "status": "passed", "description": "测试描述：错误的验证码登录失败", "steps": [{"name": "输入错误的验证码", "status": "passed", "attachments": [{"name": "错误的验证码", "source": "d06e4bbc-155f-4f21-81c1-ef2767576bc7-attachment.attach"}], "start": 1651989230784, "stop": 1651989230785}], "attachments": [{"name": "截图", "source": "4dc1d63d-06cd-47e6-a79a-50dad8bd0b17-attachment.png", "type": "image/png"}, {"name": "截图", "source": "06ccc1d3-1e9d-4016-b0f8-403f165b0a96-attachment.png", "type": "image/png"}], "start": 1651989230784, "stop": 1651989232101, "uuid": "0dd7f475-0395-45dd-a106-adefe9920ece", "historyId": "2d7cab507b6851b7595fa5868d612dc0", "testCaseId": "a971ec309964150e721c6794e448f50a", "fullName": "test_two.Test#test_23", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "登录"}, {"name": "epic", "value": "xxx测试"}, {"name": "severity", "value": "critical"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "15172-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}